### ISSUE TITLE
Extra text makes tidy difficult to use with screen readers

### ISSUE THREAD

**Issue by JohnHeim** [htacg/tidy-html5#1076]
When using tidy to check the validity of an HTML document, quite a bit of extraneous text is printed to the screen. A blind person using a screen reader has to listen to all that extra wording before finding the lines describing problems. This can waste an enormous amount of time. There should be an option to silence the extra text and just show problems/warnings.
